service/server/service/import.go

```diff
@@ -39,6 +39,7 @@ func Import(url string, which *configure.Which) (err error) {
 		strings.HasPrefix(url, "trojan-go://") ||
 		strings.HasPrefix(url, "http-proxy://") ||
 		strings.HasPrefix(url, "https-proxy://") ||
+		strings.HasPrefix(url, "socks-proxy://") ||
```

Comment:
`socks5://` is suggested.
`http-proxy` and `https-proxy` are compromises.

Reply:
> `socks5://` is suggested. `http-proxy` and `https-proxy` are compromises.

Good. I thought they were to avoid conflictions.

Reply:
And this code cause failure to import the sharing address which is exported by v2rayA.